pytest_retry/server.py

```diff
@@ -41,11 +40,12 @@ def run_server(self) -> None:
         while True:
             conn, _ = self.sock.accept()
 
-            while True:
-                chunk = conn.recv(128)
-                if not chunk:
-                    break
-                self.stream.write(chunk.decode("utf-8"))
+            with conn:
+                while True:
+                    chunk = conn.recv(4096)
```

Comment:
Why the change to the chunk size?

Reply:
To reduce the number of `recv()` calls when there is huge tracebacks in verbose mode.

Reply:
I suspect that this wouldn't make a meaningful difference (I'm open to seeing benchmarks proving otherwise, however) but I didn't see any issues with this bufsize in my research, so I guess I'm ok with it

Reply:
This isn't a "problem" per se, but tracebacks can quickly reach 1000 characters when the project gets big enough (like mine, for example).

And more generally, when it comes to sockets, the size chosen varies between 4KB and 8KB to receive the maximum amount of data with a small buffer.

(Threads on the xdist workers side send the entire traceback at once anyway).

I can do benchmarks, but it's already been shown by others that it's more efficient to do recv() with larger buffers for data processing. I recommend keeping this modification, it costs nothing.

Reply:
Yeah, like I said I think this change is fine.